international-labour-organization.csl

```diff
@@ -158,49 +169,62 @@
             </group>
           </group>
         </else-if>
-        <else-if type="chapter paper-conference" match="any">
-          <text macro="title" prefix=" " suffix="."/>
+        <else-if type="chapter" match="any">
           <group prefix=" " delimiter=" ">
-            <text term="in" text-case="capitalize-first" suffix=":"/>
+            <text variable="title" font-style="normal" suffix=","/>
+            <text term="in" text-case="lowercase" font-variant="normal"/>
             <text macro="editor"/>
             <text variable="container-title" font-style="italic" suffix="."/>
             <text variable="collection-title" suffix="."/>
+            <text variable="scale"/>
             <text macro="edition"/>
             <group suffix="." delimiter=", ">
-              <text macro="publisher"/>
+              <text macro="publisher" prefix=" "/>
               <text macro="pages"/>
             </group>
           </group>
         </else-if>
-        <else-if type="thesis">
-          <group prefix=" " delimiter=". " suffix=".">
-            <text macro="title"/>
-            <text macro="edition"/>
-            <text variable="genre"/>
-            <text macro="publisher"/>
-          </group>
-        </else-if>
-        <else-if type="webpage">
-          <group prefix=" " suffix=".">
-            <text macro="title" suffix=" [Online]."/>
-            <text macro="edition"/>
-            <text prefix=" " macro="access" suffix="."/>
-          </group>
-        </else-if>
+        <else-if type="thesis"/>
```

Comment:
I'd recommend against this -- as they currently work, Zotero and Mendeley will actually print error messages when you try to cite a thesis.